### PR TITLE
Fix hive task progress rollback issue

### LINKED ISSUE #2079: [Bug] [Linkis1.1.2]During hive task execution, the execution progress rolls back
### Search before asking

- [X] I searched the [issues](https://github.com/apache/incubator-linkis/issues) and found no similar issues.

### Linkis Component

linkis-cg-manager

### What happened + What you expected to happen

During hive task execution, the execution progress rolls back：
progress

——————————————————————————————————————————————————————
hive任务执行过程中，执行进度存在回退现象，需优化

### Relevent platform

Linkis1.1.2

### Reproduction script

DSS-Scriptis submits the following long-time hive tasks to view the execution progress：
select 
concat_ws('|',collect_list(itemlist)) as itemlist,
concat_ws('|',collect_list(servertime)) as servertime,
concat_ws('|',collect_list(device)) as device,
concat_ws('|',collect_list(itemlist)) as itemlist,
concat_ws('|',collect_list(servertime)) as servertime,
concat_ws('|',collect_list(device)) as device,
concat_ws('|',collect_list(itemlist)) as itemlist,
concat_ws('|',collect_list(servertime)) as servertime,
concat_ws('|',collect_list(device)) as device,
concat_ws('|',collect_list(recoid)) as recoid 
from ai_caixin_ads_ods_mask.recall_log limit 5000

### Anything else

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by peacewong**
### What is the purpose of the change
close #2079

### Brief change log
- Fix hive task progress rollback issue
- Add a separate ec status scan time configuration

### CODE REVIEW

**Review by wForget — APPROVED** [review]
LGTM

File: linkis-engineconn-plugins/engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala [hunk 1]
```diff
@@ -405,8 +406,12 @@ class HiveEngineConnExecutor(id: Int,
       }
 
       logger.debug(s"hive progress is $totalProgress")
-      if (totalProgress.isNaN || totalProgress.isInfinite) return currentBegin
-      totalProgress + currentBegin
+      val newProgress = if (totalProgress.isNaN || totalProgress.isInfinite)  currentBegin else totalProgress + currentBegin
```
Comment (wForget) [inline linkis-engineconn-plugins/engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala:409]: nit: extra space

Comment (peacewong) [reply linkis-engineconn-plugins/engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala]: > nit: extra space before currentBegin

I will fix this

**Comment by wForget** [comment]
Thanks, merged into master.

**Pull request merged**